src/test/fixtures/sample-local-config.json

```diff
@@ -536,6 +536,112 @@
         "allow": true,
         "accept": ".pdf, .json"
       }
+    },
+    {
+      "name": "Covering Letter (DBS)",
```

Comment:
I duplicated this from `sample-config.json` to this file because this `sample-local-config.json` (note the local) is used for our integration tests.